JavaScript_01/JavaScript_01.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="UTF-8"/>
+		<title>JAVA_01</title>
+		<script src="JavaScript_01.js"></script>
+		
+		<style>
+			body
+			{
+				text-align: center;
+			}
+			form
+			{
+				border: 2px solid black;
+				background-color:yellow;
+			}
+		</style>
+	</head>
+	<body>
+		<form method="post" action="register.php" onsubmit="return validate()">
+		<h1> LOGIN </h1>
+
+		<label for="Username">Потребителско име: </label>
+		<input type="text" id="Username" name="username"><br>
```

Comment:
You may want to add validations on the html inputs. Giving feedback to the user as early as possible is always better and this will save him time. You could check here for a reference: https://github.com/martinely95/WEB2020-php/blob/master/ex5/index.html